modules/dp_faq/dp_faq.module

```diff
@@ -29,7 +29,7 @@
 use Drupal\Core\Form\FormStateInterface;
 
 /**
- * Implements hook_form_alter().
+ * Implements hook_form_FORM_ID_alter().
  */
 function dp_faq_form_views_exposed_form_alter(array &$form, FormStateInterface $form_state, string $form_id): void {
   if ($form["#id"] === 'views-exposed-form-devportal-faq-page-1') {
```

Comment:
Could you pls also fix these double apostrophes to single ones while you're there?